services/web/src/screens/Shop/Products.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { observer, inject } from 'mobx-react';
+import { Container, Table, Message, Modal, Button, Header } from 'semantic-ui-react';
+
+import { formatDate } from 'utils/date';
+import { SearchProvider } from 'components/data';
+import HelpTip from 'components/HelpTip';
+import EditProduct from 'components/modals/EditProduct';
+
+@inject('products')
```

Comment:
this is going away as well right ? 
the stores can be singletons ?

Reply:
all API helpers are removed now and we're doing naked calls in the components... there is a single store now called `session` which is a singleton and (for now) directly imported which also means it can be imported anywhere, React component or otherwise.

I will leave some documentation in the store re: our conversation about potentially injecting these as props or... something when the need arises. Beyond that I think we should be good here.